ui/pages/confirmations/utils/confirm.ts

```diff
@@ -2,6 +2,11 @@ import { ApprovalRequest } from '@metamask/approval-controller';
 import { ApprovalType } from '@metamask/controller-utils';
 import { Json } from '@metamask/utils';
 
+export const ConfirmationRedesignTypes = [
```

Comment:
Minor, but since this is an exported constant rather than a type or enum, should we go all capitals and underscores?

Reply:
I've been confused about this myself. I've written these as all capitals and underscores in the past though noticed our repo has other patterns. 

I followed the casing from const below.

I also haven't been able to find a reference for const naming in our contributor docs 
https://github.com/MetaMask/contributor-docs/blob/main/docs/javascript.md

- 

Happy to update these to capitals and underscores since it is a common convention and this is an immutable const
ref:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const
> common convention is to use all-uppercase letters, especially for primitives because they are truly immutable.

Reply:
updated https://github.com/MetaMask/metamask-extension/pull/24182/commits/9f10833cb46939a0ec20fdd3b813a5a95cced530 &  https://github.com/MetaMask/metamask-extension/pull/24182/commits/80475d93ab25718a9d83db8bafefc4793a45e32d